### PR TITLE
Build for Java 11

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,6 +31,9 @@
         <air.check.skip-modernizer>true</air.check.skip-modernizer>
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
+
+        <project.build.targetJdk>11</project.build.targetJdk>
+        <air.modernizer.java-version>8</air.modernizer.java-version>
```
Comment (findepi) [inline pom.xml:36]: why 8 and not 11?

**Comment by findepi** [comment]
@electrum before we merge this, we should run a tempto snapshot  with Presto product tests to ensure everything on the tests side works.

As a first step, I would just have the test run on Java 11 first (with no compilation changes).
https://github.com/prestosql/presto/blob/7f2160d48a79930890df460cbd119f88baf96def/presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java#L92-L102

**Comment by electrum** [comment]
The product tests all run with Java 11 now, since the `presto-product-tests` module is built for Java 11.

**Comment by findepi** [comment]
You;re right. i got mislead by the fact we still use `centos6-oj8` as the container there.

File: pom.xml (hunk 1 above)
Comment (electrum) [reply pom.xml:36]: Updating code to newer APIs is a separate task.

**Pull request closed without merging**